newspaper/configuration.py

```diff
@@ -73,6 +74,10 @@ def __init__(self):
 
         self.thread_timeout_seconds = 1
 
+        self.ignored_content_types_defaults = {"application/pdf": self.EMPTY_PDF,
```

Comment:
Perhaps make `self.ignored_content_types_defaults` a set of `set('application/pdf', application/x-pdf', ...)` since we don't need EMPTY_PDF anymore

Reply:
Also, upon second thought I feel `self. ignored_content_types_defaults` should default to `{}`. The reason is it's not conventional / reasonable to change an API's default behaviors after many people have already downloaded and have expectations from it.

Like if we had someone using this library extensively and they expect PDFs to be returned but suddenly after the update their PDFs are all gone. It can cause a lot of headache.